File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.py
def solution(N):
    t = []
    while(N > 0):
        n = N % 10
        N = N // 10
        t.append(n)
    t = sorted(t, reverse = True)
    answer = 0
    for x in t:
        answer = answer * 10 + x
    return answer

File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.swift
func solution(_ n:Int64) -> Int64 {
    var answer : Int64 = 0
    var IntToString = String(n)
    var answerArray : [Int64] = []
    for int in IntToString {
        if let intString = Int64(String(int)) {
            answerArray.append(intString)
        }
    }
    answerArray.sort()
    answerArray.reverse()
    for index in 0..<answerArray.count {
        answer *= 10
        answer += answerArray[index]
    }
    return answer
}